packages/config/config/manufacturers.json

```diff
@@ -206,7 +206,6 @@
 	"0x00cc": "Beijing Sino-American Boyi Software Development Co. Ltd",
 	"0x00cd": "Shanghai Longchuang Eco-energy Systems",
 	"0x00cf": "Forest Group Nederland B.V",
-	"0x00d0": "HANK Electronics Ltd",
```

Comment:
Are you 100% sure about this? There are other manufacturers with multiple manufacturer IDs (Aeotec/Aeon Labs)

Reply:
I'm _not_ sure about it. I noticed that the manufacturer ID of '0x00d0' was unused in the project. I assumed that meant it was no longer needed. Could you help me understand where the source of these manufacturer codes come from? ie. How do I confirm that HANK does/doesn't have multiple manufacturer IDs?

Reply:
https://z-wavealliance.org/z-wave-specifications/ -> Registries -> Assigned Manufacturer IDs

I've used these to update our json file over the years and never deleted something unless there is a new manufacturer with the same ID. It might just be that HANK used to have `0x00d0` and there might be devices out there that use this ID.

Honestly, I prefer having one too many entries here over missing one.